requirements.txt

```diff
@@ -9,3 +9,5 @@ pyasn1
 autobahn 
 wget
 opencv-python
+Phidgets
+SimpleCV
```

Comment:
Does the Phidgets lib definitely work with Python3?  I'm pretty sure SimpleCV is Python 2.7 only and no longer maintained. I am in the process of removing the SimpleCV dependency (see octopus/image/functions.py).

Reply:
> Yes, could you please split out the re-formatting into a different PR?

Yes, sorry. I started with only reformatting but then to be sure it worked I tested it and started conflating code changes in the same commit. I'll split the two now :)

> Does the Phidgets lib definitely work with Python3? I'm pretty sure SimpleCV is Python 2.7 only and no longer maintained. I am in the process of removing the SimpleCV dependency (see octopus/image/functions.py).

The latest Phidgets API release (Phidget22) seems to support Python3.
[Phidgets22 @ Pypi]([](https://pypi.org/project/Phidget22/))

I have not tested it yet, but since we have some pressure sensors with 4..20mA connection and no reader yet I have just ordered a Phidget hub+4..20mA adapter to test it and I'll report on that in the future.

Reply:
Phidgets - cool, thanks, I look forward to the result!
SimpleCV - the project is dead so please don't add SimpleCV here. The current opencv implementation is badly documented but functional!

Reply:
> Formatting changes are unfortunately at the base of this branch, so I've moved to a different PR and I'll rebase on master to clean this once those are merged. Sorry for that.

You should be able to do a rebase to separate the two branches - [see the howto here](https://gist.github.com/unbracketed/889c844473bcca1917e2).